lib/launch-video.js

```diff
@@ -13,7 +13,7 @@ function launchVideo(opts, panel) {
   let id;
 
   // TODO: submit a fix to getVideoId for this. #226
-  if (opts.url.indexOf('attribution_link')) {
+  if (opts.url.indexOf('attribution_link') > -1) {
```

Comment:
I see some unrelated extra stuff in this PR. Are you just doing some unrelated cleanup work, or were these youtube-related changes unintentionally added to the commit?